File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int n=nums.length;
        Set<Integer> mp= new HashSet<>();
        for(int i=0;i<n;i++)
        {
            mp.add(nums[i]);
        }
        
        int res=0;
        for(int i=0;i<n;i++)
        {
            int count=0;
            if(mp.contains(nums[i]-1))
                continue;
            int num=nums[i];            
            while(mp.contains(num))
            {
                count++;
                num++;
                res=Math.max(res,count);
            }
            res=Math.max(res,count);
        }
        return res;        
    }

}

File: Data Structures & Algorithms/maximum-subarray/submission-1.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:

        
        max_so_far = float("-inf")
        max_ending = 0
        
        for num in nums:
            max_ending = max(num,max_ending + num)
            max_so_far = max(max_so_far,max_ending)
        return max_so_far